tsconfig.prod.json

```diff
@@ -1,23 +1,24 @@
 {
+  "extends": "@tsconfig/node10/tsconfig.json",
```

Comment:
If this is going to be the code that people are going to use, let's specify a target of `es5` so that WebdriverIO will keep working with older browsers as well 👍

PS: I believe it's `es5` by default but I think it's good to make it explicit for visibility and in order to prevent issues if the default ever changes

Reply:
> so that WebdriverIO will keep working with older browsers as well 👍

This code is not run in the browser 😛  WebdriverIO will continue to work given that automation is not run by JS.

Reply:
But some commands are injected into the browser and if we output es6 or above it will break unless I am missing something

Reply:
None of the scripts have been targeted to ES5 when [Babel was around](https://github.com/webdriverio/webdriverio/blob/v6.3.7/babel.config.js). None of them use some specific ESNext functionality. I feel confident that they continue to work even if testing on IE6.